yb_migrate/src/migration/postgres.go

```diff
@@ -33,28 +33,6 @@ import (
 
 var commandNotFoundRegexp *regexp.Regexp = regexp.MustCompile(`(?i)not[ ]+found[ ]+in[ ]+\$PATH`)
 
-// TODO: check for pgdump and psql/ysqlsh - installed/set-in-path
-func CheckToolsRequiredForPostgresExport() {
```

Comment:
Why this code was required to be removed for Blob removal capability?

Reply:
This function is stale/not-used now. Instead we are using `CheckToolsRequiredInstalledOrNot()` everywhere.